pages/index.js

```diff
@@ -245,6 +246,50 @@ export default function Home({
                               </StyledForm>
                             </>
                           )}
+                          {addAccount === "Stocks account" && (
+                            <>
+                              <StyledForm>
+                                <StyledFieldset>
+                                  <label>
+                                    {`Stock name    `}
+                                    <StyledInput type="text" />
+                                  </label>
+                                  <label>
+                                    {`WKN    `}
+                                    <StyledInput
+                                      type="text"
+                                      style={{width: "3rem"}}
```

Comment:
are you mixing inline styles and styled components? if you already have a StyledInput, can you transfer the width there? this would apply to other input elements as well. Alternatively, if you need a StyledInput element without the width specification, I would suggest creating two styled components like this: 
```const WideStyledInput = styled(StyledInput)`width: 3rem;` ```

Reply:
Hi Ulrike, I have a styled component `StyledInput` which I am using it in several places but I need it to be different sizes, hence the inline styling. To have another styled component based on `StyledInput` is also a solution, but for just one property I preferred inline styling. Might implement your suggestion though for other styled components where I have more properties for inline styling. Thank you. 🙏